frontend-react/src/styles/_components.scss

```diff
@@ -1,4 +1,6 @@
 $theme-hr-border-color: "base-lighter";
+$bg-col-header: #73B3E7;
```

Comment:
Can we use `color("primary-light")` and `color("primary-lighter")` instead of introducing new hardcoded hex codes?

Reply:
We actually have access to `$blue-10` and `$blue-30` now! Will change.